lib/services/ngsi/entities-NGSI-v2.js

```diff
@@ -323,6 +324,7 @@ function sendQueryValueNgsi2(entityName, attributes, typeInformation, token, cal
  */
 function sendUpdateValueNgsi2(entityName, attributes, typeInformation, token, callback) {
     let payload = {};
+    const explicitAttrsList = ['TimeInstant'];
```

Comment:
Maybe is a good idea to move this part where all the timeinstant is managed (even if this part of the code needs to be refactored in the future): 

Here:
https://github.com/telefonicaid/iotagent-node-lib/blob/ddbcd33094e64003da4cacfd0c2743cd723e437f/lib/services/ngsi/entities-NGSI-v2.js#L392-L406

And here:
https://github.com/telefonicaid/iotagent-node-lib/blob/ddbcd33094e64003da4cacfd0c2743cd723e437f/lib/services/ngsi/entities-NGSI-v2.js#L428-L440

Further, this is declared as a "magic string". It is not a good pattern to follow. Using the constant defined for this purpose is a better idea:

https://github.com/telefonicaid/iotagent-node-lib/blob/6748fc9cedee6133d530e7fd37581236e8d369ec/lib/constants.js#L56

Reply:
Magic number replaced by constant - Fixed b707d37